### ISSUE TITLE
Able to Reply to Spam and Unapproved Comments as Comment Admin

### ISSUE THREAD

**Issue by gordonbanderson** [silverstripe/silverstripe-comments#170]
The current check, pasted below, does not take into account the moderation or spam status of the comment.  Whilst only an admin should see this it makes sense for them not to be replied to.

```php
    public function getRepliesEnabled() {
		// Check reply option
		if(!$this->getOption('nested_comments')) {
			return false;
		}

		// Check if depth is limited
		$maxLevel = $this->getOption('nested_depth');
		return !$maxLevel || $this->Depth < $maxLevel;
	}
```

**Comment by gordonbanderson** [comment]
Makes more sense now I think
![nospamreply](https://cloud.githubusercontent.com/assets/7060/13028205/d86e1f54-d29a-11e5-9b49-2230810ac082.png)

**Comment by tractorcow** [comment]
Right that's a good idea. :D

**Comment by tractorcow** [comment]
Fixed with #172
